fe/fe-core/src/main/java/com/starrocks/sql/optimizer/cost/CostModel.java

```diff
@@ -329,14 +315,15 @@ public CostEstimate visitPhysicalCTEProduce(PhysicalCTEProduceOperator node, Exp
 
         @Override
         public CostEstimate visitPhysicalCTEAnchor(PhysicalCTEAnchorOperator node, ExpressionContext context) {
-            return CostEstimate.zero();
+            // memory cost
```

Comment:
This cost why not compute at CTE producer?

Reply:
of course if you want